crates/pacdef_core/src/backend/actual/fedora.rs

```diff
@@ -0,0 +1,127 @@
+use core::panic;
+use std::collections::HashSet;
+use std::process::Command;
+
+use anyhow::Result;
+
+use crate::backend::backend_trait::{Backend, Switches, Text};
+use crate::backend::macros::impl_backend_constants;
+use crate::cmd::run_external_command;
+use crate::{Group, Package};
+
+#[derive(Debug, Clone)]
+pub struct Fedora {
+    pub(crate) packages: HashSet<Package>,
+}
+
+const BINARY: Text = "dnf";
+const SECTION: Text = "fedora";
+
+const SWITCHES_INFO: Switches = &[
```

Comment:
BTW, `SWITCHES_INFO` was added for `pacdef package review`. When you want more information about the package you are currently reviewing, you could hit `i`, and it would print something like this (Arch Linux in this case):

```
Name            : linux
Version         : 6.8.4.arch1-1
Description     : The Linux kernel and modules
Architecture    : x86_64
...
```

It's possible I have missed that in the PR for rustup.